src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -308,6 +309,25 @@ public static Priority parseTaskPriority(String priority) throws ParseException
         }
     }
 
+    /**
+     * Parses a {@code String status} into a {@code Status}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code status} is invalid.
+     */
+    public static Status parseTaskStatus(String status) throws ParseException {
+        assert status != null;
+        String priorityAllUpperCase = status.toUpperCase();
+        switch(priorityAllUpperCase) {
+        case("COMPLETED"):
+            return Status.COMPLETED;
+        case("INCOMPLETE"):
+            return Status.NOT_COMPLETED;
+        default:
```

Comment:
Yup this is a nice change. Perhaps I should have used "INCOMPLETE" from the beginning hahaha 😆 I chose to use `NOT_COMPLETED` to make the command name for intuitive, but I decided to name the command "resettask" instead. So, using `INCOMPLETE` shouldn't have been a problem 😅

Reply:
Thanks for the note!